BDArmory/Modules/ModuleWeapon.cs

```diff
@@ -2184,7 +2201,12 @@ private void LaserBeam(string vesselname)
                     tracerEndWidth = Mathf.Lerp(tracerEndWidth, grow ? 1 : 0.05f, 0.35f); //add new tracerGrowWidth field?
                     if (grow && tracerStartWidth > 0.95) grow = false;
                     if (!grow && tracerStartWidth < 0.06f) grow = true;
-                    UpdateLaserSpecifics(true, false, false, true);
+                    UpdateLaserSpecifics(true, dynamicFX, true);
+                }
+                //if (cycleTexture)
+                if (dynamicFX)
+                {
+                    UpdateLaserSpecifics(false, true, false);
```

Comment:
This could probably be removed - the call on 3595 should already be covering updating FX, making this superfluous

Reply:
fair enough, yeah, it looks like it could be removed.